config_vars.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# API keys
+export MICROSOFT_TRANSLATE_API_KEY=''
+export GOOGLE_TRANSLATE_API_KEY=''
+export WORDNIK_API_KEY=''
+
+# Set to YES if running in the cloud
+RUNNING_IN_CLOUD='NO'
+
+# Trying to detect if running in a cloud...
+curl metadata.google.internal -si 2>&1>/dev/null
+if [ $? -eq 0 ]; then
+    RUNNING_IN_CLOUD='YES'
+fi
+curl http://169.254.169.254/1.0/meta-data/instance-id -s 2>&1>/dev/null
```

Comment:
what's this hardcoded IP address doing here?

Reply:
It's the cloud instance metadata URL. It's what AWS uses to pass metadata to the instances created in the cloud. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
It helps detecting if the script is being run on an instance from inside a cloud.